lib/app.js

```diff
@@ -50,6 +50,57 @@ app.get('/discs/:brand', async(req, res) => {
   }
 });
 
+app.post('/discs/', async(req, res)=>{
+  try {
+    const data = await client.query(`
+      INSERT INTO discs(
+        disc,
+        speed,
+        type,
+        brand,
+        stable,
+        plastics
+      ) VALUES ($1, $2, $3, $4, $5, $6)
+      RETURNING *;`, [
+      req.body.disc,
+      req.body.speed,
+      req.body.type,
+      req.body.brand,
+      req.body.stable,
+      req.body.plastics
+    ]);
+    res.json(data.rows[0]);
+  }catch(e){
+    res.status(500).json({ error: e.message });
+  }
+});
+app.put('/discs/:id', async(req, res)=>{
+  try {
+    const data = await client.query(`
+    UPDATE discs
+    SET
```

Comment:
Great job with the SET! Fun fact, if you use `.patch` instead of `.put`, it does the same update but you only have to send along the piece of info that you want to update rather than including every single property of the object. Ref: https://rapidapi.com/blog/put-vs-patch/